RawSpeed/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 find_package(JPEG REQUIRED)
 include_directories(SYSTEM ${JPEG_INCLUDE_DIRS})
 
+find_package(ZLIB REQUIRED)
```

Comment:
@houz is right. But now, it is up to me i guess.
I consider zlib to be ok as a dep. I may need to revisit this once i finally start to look at fuzzing, but for now it is ok.

What i do however think, is that it should be an *option*, i.e. it should be able to pass `-DWITH_ZLIB=OFF` to cmake, and that code will not be compiled.